flexmeasures/data/models/planning/storage.py

```diff
@@ -142,12 +144,43 @@ def compute(
                 resolution,
             )
 
-        device_constraints[0]["min"] = (soc_min - soc_at_start) * (
-            timedelta(hours=1) / resolution
+        soc_min_change = (soc_min - soc_at_start) * timedelta(hours=1) / resolution
+        soc_max_change = (soc_max - soc_at_start) * timedelta(hours=1) / resolution
+
+        if soc_minima is not None:
+            device_constraints[0]["min"] = build_device_soc_targets(
```

Comment:
I like the idea of keeping the old syntax in tests, so that we are reminded of the deprecation when running our test suite, but in this case we would also see deprecation warnings in server logs, which I find undesirable.